insights/specs/insights_archive.py

```diff
@@ -31,6 +31,7 @@ class InsightsArchiveSpecs(Specs):
     certificates_enddate = first_file(["insights_commands/find_.etc.origin.node_.etc.origin.master_.etc.pki_.etc.ipa_-type_f_-exec_.usr.bin.openssl_x509_-noout_-enddate_-in_-exec_echo_FileName", "insights_commands/find_.etc.origin.node_.etc.origin.master_.etc.pki_-type_f_-exec_.usr.bin.openssl_x509_-noout_-enddate_-in_-exec_echo_FileName"])
     chkconfig = simple_file("insights_commands/chkconfig_--list")
     chronyc_sources = simple_file("insights_commands/chronyc_sources")
+    cloud_init_network = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_cloud_init_network")
```

Comment:
I don't think this is required for @datasource specs and it's not necessary to add it to the uploader.json as well. the `insights-cat` in uploader.json is just a temporary way to collect some datasources which are for 2019 summit. @bfahr - how do you think?

Reply:
@xiangce that is correct.  This spec will be stored as a file spec in `/etc/cloud/cloud.cfg` in the archive, based on the `relative_path` argument provided in the return statement of the `datasource`.  Please remove this change @psachin

Reply:
This is new information for me. I was under the impression that the collected data will be stored in `insights_commands/python_-m_insights.tools.cat_--no-header_cloud_cfg`. But I think it is more appropriate to collect the data in `/etc/cloud/cloud.cfg`. 

I'll remove this